CONTRIBUTING.md

```diff
@@ -39,26 +39,26 @@ We cover that briefly in the next section, and in more detail in [Developing wit
 
 ## Building your own plugins and interfaces
 
-You don't need approval from existing QIIME 2 developers or maintainers to create and distrubute your own tools, such as plugins, that build on QIIME 2.
-We encourage you to build your own.
+You don't need approval from existing QIIME 2 developers or maintainers to create and distrubute your own tools, such as plugins, that build on QIIME 2. This is the beauty of the plugin-based architecture of QIIME 2, that development of tools for introducing novel functionality can be distributed, and individual QIIME installations can be customized by users.
```

Comment:
```suggestion
You don't need approval from existing QIIME 2 developers or maintainers to create and distribute your own tools, such as plugins, that build on QIIME 2.
This is the beauty of the plugin-based architecture of QIIME 2: development of tools for introducing novel functionality can be distributed by anyone, removing the lead development team as a bottleneck or gatekeeper between developers of new functionality and users interested in that functionality, and customized QIIME 2 distributions can be created and shared by users.
```

Made a couple of edits here, and modified to have one sentence per line (which makes text documents more management with respect to pull request reviews -- in markdown the first newline character is ignored, two are needed for a paragraph break).